src/components/pages/batch/list/index.tsx

```diff
@@ -56,27 +58,22 @@ function BatchListPageComponent() {
   };
 
   const handleOnCreateLot = () => {
-    const ccIds = [...new Set(selectedBatches.map((b) => b.ccCode))];
-    const prefix = ccIds.length > 1 ? co.label : ccs.find((c) => c.value === ccIds[0]).label;
+    const prefix = "Mityana";
     const quantity = selectedBatches.reduce(
-      (acc, cv) =>
-        (selectedBatches.length && selectedBatches[0].type === BATCH_TYPE.DRY
-          ? cv.quantity
-          : cv.perchmentQuantity) + acc,
+      (acc, cv) => selectedBatches.length && cv.quantity + acc,
       0
     );
+
     const payload = {
       name: `${prefix}_${selectedBatches[0].type.charAt(0).toUpperCase()}_`,
       type: selectedBatches[0].type,
       selected: selectedBatches,
-      coCode: co.value,
+      coCode: "71", //TODO: change it to co.value
```

Comment:
need to verify if it should be hard coded to 71 or get from co.value as comment in the code suggests